wrappers/C_CPP/README.md

```diff
@@ -1,18 +1,40 @@
 # REFPROP wrapper for C/C++
 
-Brought to you by Ian Bell, NIST, 2016
-
-Help!
------
-
-Please file an issue at https://github.com/usnistgov/REFPROP-issues and we will get back to you as quickly as possible.  Please don't email NIST staff directly, the only exception being if you have proprietary code that cannot be shared publicly.
-
 Installation
 ------------
+**NOTE**: for Linux and macOS, you must compile REFPROP yourself, see: [REFPROP-cmake](https://github.com/usnistgov/REFPROP-cmake)
+
+1. Follow the conventional installation instructions for REFPROP.
+2. For Linux and macOS, compile REFPROP via [REFPROP-cmake](https://github.com/usnistgov/REFPROP-cmake).
+3. Ensure that the system environment variable `RPprefix` is set to the location of the REFPROP installation (e.g., `C:\Program Files (x86)\REFPROP`).
```

Comment:
Not on Windows, but they are on Linux and macOS. Since this is the case of the variable created by the REFPROP installation utility, I think it best to be consistent with that to avoid confusion or any issues.

Reply:
This was news to me, but it seems you are right. On windows environment variables do not appear to be case-sensitive. Dunno about others.